style.css

```diff
@@ -144,83 +130,414 @@ li {
+  padding: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
 
-  /* Inside auto layout */
+.java-icon p {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  color: #253858;
+}
 
+.java-icon {
+  background: #f7f7f9;
+  border-radius: 8px;
+  padding-left: 12px;
+  padding-top: 10px;
+  padding-right: 35;
+  padding-bottom: 48;
   flex: none;
-  order: 1;
+  order: 0;
   flex-grow: 0;
-  padding-left: 16px;
-  padding-right: 16px;
+}
+
+.html-icon {
+  background: #f7f7f9;
+  border-radius: 8px;
+  padding-left: 12px;
+  padding-top: 10px;
+  padding-right: 35;
+  padding-bottom: 48;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.html-icon p {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  color: #253858;
+}
+
+.css-icon {
+  background: #f7f7f9;
+  border-radius: 8px;
+  padding-left: 12px;
+  padding-top: 10px;
+  padding-right: 35;
+  padding-bottom: 48;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.css-icon p {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  color: #253858;
+}
+
+.lan-tools {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  gap: 1.5rem;
+}
+
+.about-my-self p {
+  font-size: 16px;
+  color: #344563;
+  margin: 0;
+}
+
+#frame p {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #344563;
+}
+
+.last-part {
+  background-color: #396df2;
+}
+
+.lan-desc {
+  flex-direction: column;
+  height: 24px;
+  left: 24px;
+  right: 78px;
+  top: calc(50% - 24px / 2);
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #344563;
+}
+
+.btn:enabled {
+  color: #6070ff;
+  border: 1px solid #7f8cff;
+  border-radius: 8px;
+}
+
+.btn:hover {
+  background: #6070ff;
+  color: #fff;
+}
+
+.btn:disabled {
+  background-color: white;
+  color: #5e6c84;
+  border: 1px solid black;
+}
+
+.btn:visited {
+  background: #2230d2;
+  color: #fff;
+}
+
+.about-my-self {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0 24px;
+  border-top-right-radius: 70px;
+  background-color: #fff;
+}
+
+.about-my-self .top {
+  margin-top: 114px;
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
+
+.about-my-self h1 {
+  font-size: 40px;
+  color: #172b4d;
+  margin: 0;
+}
+
+.social-media .connect {
+  color: #7f8cff;
+}
+
+.btn.Get-my-resume {
+  background-color: #fff;
+  align-self: flex-start;
+}
```

Comment:
- [ ] Good work implementing the hover in your Get my resume button ✔️. However, please ensure that the background color on hover is blue. 🤓 Currently, the background color on hover is white.

You can see how it looks below 👇🏽

https://user-images.githubusercontent.com/116701485/236592899-6563fdbc-de2f-4d1d-b81e-d66fc937a77e.mp4